udata_gouvfr/settings.py

```diff
@@ -9,7 +9,8 @@
 
 # Frontend banner parameters
 BANNER_ACTIVATED = False
-BANNER_HTML_CONTENT = ''
+EN_BANNER_HTML_CONTENT = ''
+FR_BANNER_HTML_CONTENT = ''
```

Comment:
```suggestion
BANNER_HTML_CONTENT_FR = ''
BANNER_HTML_CONTENT_EN = ''
```

In order to have a consistent `BANNER_` prefix.